### PR TITLE
Feature/log stream outputs

### PR DESCRIPTION
**Pull request by abulford**
Adds `aws_partner_event_source` and `azure_partner_topic` as computed outputs to the log stream resource, as discussed in the [original log stream PR](https://github.com/alexkappa/terraform-provider-auth0/pull/270).

### CODE REVIEW

File: auth0/resource_auth0_log_stream.go [hunk 1]
```diff
@@ -164,6 +176,8 @@ func readLogStream(d *schema.ResourceData, m interface{}) error {
 	d.Set("status", c.Status)
 	d.Set("type", c.Type)
 	d.Set("sink", flattenLogStreamSink(d, c.Sink))
+	d.Set("aws_partner_event_source", logStreamAwsPartnerEventSource(c.Sink))
```
Comment (mcalster) [inline auth0/resource_auth0_log_stream.go:179]: @abulford  this code is already implemented in flattenLogStreamEventGridSink and flattenLogStreamEventBridgeSink respectively.

File: auth0/resource_auth0_log_stream.go [hunk 2]
```diff
@@ -132,6 +132,18 @@ func newLogStream() *schema.Resource {
 					},
 				},
 			},
+			"aws_partner_event_source": {
```
Comment (mcalster) [inline auth0/resource_auth0_log_stream.go:135]: I think this omission is why you don't get the partner topic back?

Comment (abulford) [reply auth0/resource_auth0_log_stream.go:135]: @mcalster yes, adding this into the `sink` object does return the values, but they're not easily accessible.  I've gone into more detail in your PR to the upstream repo.

**Comment by mcalster** [comment]
All the changes have been discussed and implemented in master. @abulford Thx for contributing!

**Pull request closed without merging**